release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2995,6 +2995,57 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.4
+[id="ocp-4-18-4_{context}"]
+=== RHSA-2025:2449 - {product-title} {product-version}.4 bug fix update and security update
+
+Issued: 11 March 2025
+
+{product-title} release {product-version}.4 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:2449[RHSA-2025:2449] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:2451[RHBA-2025:2451] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.4 --pullspecs
+----
+
+[id="ocp-4-18-4-known-issues_{context}"]
+==== Known issues
+
+* There is currently a known issue where a Technology Preview-enabled cluster has Sigstore verification for payload images in `policy.json`, but the Podman version in the base image does not support Sigstore configuration, so the new node is not available.
++
+Workaround: The node will start running when the Podman version in the base image does not support Sigstore. Use the default `policy.json` that does not have Sigstore verification if the base image is 4.11 or earlier. (link:https://issues.redhat.com/browse/OCPBUGS-48296[*OCPBUGS-48296*])
```

Comment:
I suggest using a noun here and in the previous paragraph to define what `policy.json` means. For example, "the `policy.json` file".